docs/get_started/plan_deployment/traditional_deployment/plan_multiple_profile.md

```diff
@@ -1,47 +1,47 @@
-# Multiple profile support
+# Multiple-profile support
 
-Multiple profiles give you the ability to have multiple, independently configured portal instances that run from the same installation.
+With multiple profiles you can work with multiple, independently configured portal instances that run on the same installation.
 
-Starting with Portal Version 7, it is possible to create extra profiles in addition to the default profile, which is useful in the following scenarios:
+Starting with Portal Version 7, you can create extra profiles in addition to the default profile. Extra proviles are useful in the following scenarios:
```

Comment:
profiles spelling is wrong:
Change "Extra proviles are useful in the following scenarios:" to
"Extra profiles are useful in the following scenarios:"

Reply:
Thanks.

Reply:
Yes Philips you have to change in file.